int64 pose_key_i
int64 vel_key_i
int64 pose_key_j
int64 vel_key_j
int64 bias_key
float32[3] accel_bias
float32[3] gyro_bias
string serialized_pim